src/test/resources/Features/OCD-2806-SimplifiedEditingOfVersion.feature

```diff
@@ -45,3 +45,18 @@ Scenario: Verify error message is displayed when the version field is left blank
     And I set the version field to ""
     Then I see error message "Field is required" for version "54"
 
+Scenario Outline: User logs in as ROLE and sees version split link exists on new developer page
+    Given I'm logged in as "<ROLE>"
```

Comment:
If you have multiple examples, then all of the variables should be in the table. In the other Scenario in this PR there's only one example, so not having the table is fine